server/odc-service/src/main/java/com/oceanbase/odc/service/task/config/TaskFrameworkConfiguration.java

```diff
@@ -95,15 +108,48 @@ public TaskFrameworkEnabledProperties taskFrameworkEnabledProperties(
     }
 
     @Bean
+    @ConditionOnServer
     public JobConfiguration jobConfiguration() {
         return new DefaultSpringJobConfiguration();
     }
 
     @Bean
+    @ConditionalOnBean(JobConfiguration.class)
     public JobSchedulerFactoryBean jobSchedulerFactoryBean(@Autowired JobConfiguration jobConfiguration) {
         JobSchedulerFactoryBean factoryBean = new JobSchedulerFactoryBean();
         factoryBean.setJobConfiguration(jobConfiguration);
         return factoryBean;
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    public JobScheduler jobScheduler() {
```

Comment:
why add a empty implementation for JobScheduler

Reply:
Avoid bean loading failures that depend on JobScheduler.